.github/workflows/stale_issues.yml

```
@@ -7,33 +7,42 @@ permissions:
   contents: read
 
 jobs:
-  stale:
+  stale-prs: # close stale PRs after ~3 months
     permissions:
-      issues: write  # for actions/stale to close stale issues
-      pull-requests: write  # for actions/stale to close stale PRs
+      pull-requests: write
     runs-on: ubuntu-latest
     steps:
       - uses: actions/stale@v3
         with:
-          stale-issue-message: '👋 Hey there. This issue hasn''t had any activity for 180 days. We''ll automatically close it if that trend continues for another week. If you feel this issue is still valid and needs attention please let us know with a comment.'
           stale-pr-message: '👋 Hey there. This PR hasn''t had any activity for 90 days. We''ll automatically close it if that trend continues for another week. If you feel this issue is still valid and needs attention please let us know with a comment.'
-          close-issue-message: '❌ We''re automatically closing this issue due to lack of activity. Please comment if you feel this was done in error.'
           close-pr-message: '❌ We''re automatically closing this PR due to lack of activity. Please comment if you feel this was done in error.'
           days-before-pr-stale: 90
-          days-before-issue-stale: 180
           days-before-close: 7
-          stale-issue-label: 'stale-issue'
-          close-issue-label: 'stale-issue-closed'
-          exempt-issue-labels: 'bug,skip-stale-check,meta'
           stale-pr-label: 'stale-pr'
           close-pr-label: 'stale-pr-closed'
-          exempt-pr-labels: 'skip-stale-check,meta'
-          exempt-all-assignees: true
           operations-per-run: 1000
           remove-stale-when-updated: true
           enable-statistics: true
           debug-only: false
-  answered: # automatically close issues labeled as 'answered' if there was no response in 3 days
+  stale-issues: # close stale issues after ~1 year
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          stale-issue-message: '👋 Hi there - this issue hasn''t had any activity in 6 months. If the EUI team has not explicitly expressed that this is something on our immediate roadmap, it''s unlikely that we''ll pick this issue up. We would sincerely appreciate a PR/community contribution if this is something that matters to you! If not, and there is no further activity on this issue for another 6 months (i.e. it''s stale for over a year), the issue will be auto-closed.'
```

Old message for reference:

> 👋 Hey there. This issue hasn''t had any activity for 180 days. We''ll automatically close it if that trend continues for another week. If you feel this issue is still valid and needs attention please let us know with a comment.

What this new message attempts to convey:

- If we're not working on it, the other option is a community contribution/PR
- If neither of the above happens, the issue will auto close after being open for a year

I deliberately removed the `If you feel this issue is still valid` text - just because a request is valid, doesn't mean it is has priority or is something we have bandwidth to do. As a reminder, we already auto-skip the `bug` label so in terms of 'validity', fixes are considered always valid.

---

I think this message has the right information and objectivity. Is "immediate" used to mean in the next year WRT our roadmap? Would that statement be more direct by saying "...that this is something on our roadmap, it's unlikely that we'll pick this issue up."

---

Hmm, yeah, I think I meant "immediate" to mean in the next year. I guess we don't (currently) have roadmaps further out than a year though, so we could def just nuke that adjective!

```suggestion
          stale-issue-message: '👋 Hi there - this issue hasn''t had any activity in 6 months. If the EUI team has not explicitly expressed that this is something on our roadmap, it''s unlikely that we''ll pick this issue up. We would sincerely appreciate a PR/community contribution if this is something that matters to you! If not, and there is no further activity on this issue for another 6 months (i.e. it''s stale for over a year), the issue will be auto-closed.'
```

---

I like the new message 👍